tests/codegen/fcode/scripts/imports.py

```diff
@@ -4,11 +4,12 @@
 #
 #from expressions import ad, bd
 #xd = ad * bd
+from numpy import ones
 xi = 2
 
 from decorators_types import decr
 yi = decr(xi)
 
 from classes_2 import Point
 
-p = Point(0.0)
+p = Point(ones(4))
```

Comment:
This is related to #1586 . I think it would be better to avoid this complication. Can `classes_2.py` be modified to fix this?

Reply:
Never mind, the scope of the Dummy variable that would be created is the same as the scope of `p` so you should be ok here.